### PR TITLE
Enable CUD operations for layers

### PR DESCRIPTION
**Pull request by dnlkoch**
This adds a `LayerEditForm` to enable creation, updating and deletion of layers.

Please review @terrestris/devs.

### CODE REVIEW

File: src/Component/Layer/LayerEditForm/LayerEditForm.tsx [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+    const updates = (await form.validateFields()) as Layer;
+    const updatedLayer = {
+      ...layer,
+      ...updates
+    };
+
+    const updateMode = id.toString() !== 'create';
+    const name = updatedLayer.name;
+
+    console.log(updateMode);
```
Comment (hwbllmnn) [inline src/Component/Layer/LayerEditForm/LayerEditForm.tsx:118]:
```suggestion
```

**Review by hwbllmnn — APPROVED** [review]

**Comment by dnlkoch** [comment]
Thanks for the review.

File: src/Component/Layer/LayerTypeSelect/LayerTypeSelect.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    setCurrentValue(value);
+
+    onChange(value);
+  };
+
+  return (
+    <Select
+      defaultValue={currentValue}
+      value={currentValue}
+      options={getOptions()}
```
Comment (KaiVolland) [inline src/Component/Layer/LayerTypeSelect/LayerTypeSelect.tsx:63]: Minor: Calling a function here is not needed when using a FC. You could just use the options array and omit the function.

Comment (dnlkoch) [reply src/Component/Layer/LayerTypeSelect/LayerTypeSelect.tsx:63]: True, but the `options` must be created/filled beforehand and I like to have this encapsulated in a function.

**Pull request merged**